packages/admin/src/routes/entities/components/RelationsView/RelationsView.js

```diff
@@ -77,13 +81,13 @@ const RelationsView = ({
               {relation.relationDisplay.label}</RelationLabel>{getRelationCountLabel(relation.relationName)}
             <RelationLinks>
               <StyledLink
-                aria-label="go to relation list view"
+                aria-label={msg('client.admin.entities.relationsView.relationLinkView')}
```

Comment:
if we go through the hassle to adding text resources for aria-labels i think we should us them as title as well (for mouseover). what du you think?